utils.go

```diff
@@ -97,14 +97,6 @@ func reverseOrientation(img image.Image, o string) *image.NRGBA {
 	case "8":
 		return imaging.Rotate90(img)
 	}
-	logrus.Errorf("unknown orientation %s, expect 1-8", o)
+	logrus.Errorf("unknown orientation %s, expect 2-8", o)
```

Comment:
It seems we didn't log any errors before when orientation was 1. Shouldn't we keep it like that?
```suggestion
	logrus.Errorf("unknown orientation %s, expect 1-8", o)
```

Reply:
Yeah, but the  `if orient.String() == "1" { return &imgBody }` part at the top is supposed to return early without much processing if orientation is 1, so this log shouldn't show up in that case, should i change it to 1-8 anyway?

Reply:
Not a big deal, then. The error message will be wrong if the function is called from another context, but right now it's fine.